MovieQuiz/Services/StatisticService.swift

```diff
@@ -57,8 +57,8 @@ final class StatisticServiceImplementation: StatisticService {
     
     func store(correct count: Int, total amount: Int) {
         let newGame = GameRecord (correct: count,
```

Comment:
здесь пропустила в прошлый раз, перед круглой обычно пробел в функциях не оставляют
```suggestion
        let newGame = GameRecord(correct: count,
```